src/daekyeong/ManagementSystemImpl.java

```diff
@@ -0,0 +1,81 @@
+package daekyeong;
+
+import java.util.*;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class ManagementSystemImpl implements ManagementSystem {
+
+    HashMap<Integer, Member> member = new HashMap<>();
+
+    @Override
+    public boolean addMember(String name, int age, int id) {
+        if(member.containsKey(id)) {
+            return false;
+        }
+        else {
+            member.put(id, new Member(name, age, id));
+            return true;
+        }
+    }
```

Comment:
```suggestion
    @Override
    public boolean addMember(String name, int age, int id) {
        if (member.containsKey(id)) {
            return false;
        }
        member.put(id, new Member(name, age, id));
        return true;
    }
```

`else`문 없이 `early return` 구조를 사용하면 더 좋은 코드가 될 것 같아요!
아래 블로그를 참고해 보세요~
https://limdingdong.tistory.com/8